src/main/java/io/jenkins/plugins/models/PluginRelease.java

```diff
@@ -0,0 +1,36 @@
+package io.jenkins.plugins.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Date;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class PluginRelease {
+  @JsonProperty("tag_name") final private String tagName;
+  @JsonProperty("name") final private String name;
+  @JsonProperty("published_at") final private Date publishedAt;
+  @JsonProperty("body") private String body;
+
+  public PluginRelease() {
+    this("", "", new Date(), "");
```

Comment:
What’s with these empty strings? Do you need a default constructor?

If it’s Jackson you can annotate the other constructor with JsonCreator

Reply:
Done